FhirToDataLake/azurepipelines/ci.yml

```diff
@@ -0,0 +1,106 @@
+steps:
+- task: NuGetAuthenticate@0
+  displayName: 'NuGet Authenticate'
+
+- task: NuGetToolInstaller@1
+  displayName: 'Use NuGet 5.11'
+  inputs:
+    versionSpec: 5.11
+
+# Install for Azure Function dependencies
+- task: UseDotNet@2
+  displayName: 'Install .NET Core sdk 3.1'
```

Comment:
Also needs to upgrade to `.NET 6.0`?

Reply:
This is just for building function app dependencies in build agent. Will remove when we deprecate the functionapp project